source/developers/hassio/addon_config.markdown

```diff
@@ -135,7 +138,9 @@ The `options` dict contains all available options and their default value. Set t
     { "username": "cheep", "password": "654321" }
   ],
   "random": ["haha", "hihi", "huhu", "hghg"],
-  "link": "http://blebla.com/"
+  "link": "http://blebla.com/",
```

Comment:
Please use `example.com` for documentation purposes. This is the official domain reserved for this by the IANA and described in RFC 2606.